libraries/stdlib/jdk8/src/kotlin/random/jdk8/PlatformThreadLocalRandom.kt

```diff
@@ -8,15 +8,20 @@ package kotlin.random.jdk8
 import java.util.concurrent.ThreadLocalRandom
 
 @Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER", "CANNOT_OVERRIDE_INVISIBLE_MEMBER")
-internal class PlatformThreadLocalRandom : kotlin.random.AbstractPlatformRandom() {
+internal object PlatformThreadLocalRandom : kotlin.random.AbstractPlatformRandom(), Serializable {
+    private const val serialVersionUID = -8182665159126504406L
```

Comment:
Why such `serialVersionUID` values, do they mean something? I believe we can start with `serialVersionUID = 1L` for all serializable entities.

Reply:
I used JDK [serialver](https://docs.oracle.com/javase/7/docs/technotes/tools/solaris/serialver.html) utility that creates something like hash, which has to be changed if the class's structure is changed